docs/source/initialsetup/mameintro.rst

```diff
@@ -1,43 +1,73 @@
+2. to read the corresponding **sysinfo.dat** entry in order to better
+understand which issues you may encounter while running a system in MAME
+(again, for Apple Macintosh Plus you have to check this entry).
 
-Alternatively, you can simply see the status by yourself, launching the system emulation and taking a look to the red or yellow warning screen which appears before the emulation starts, if any. Notice that if you have information which can help to improve the emulation of a supported system, or if you can directly contribute fixes and/or addition to the current source, you can follow the instructions at the contact page or post to the MAME Forums at http://forum.mamedev.org/
+Alternatively, you can simply see the status by yourself, launching the system
+emulation and taking a look at the red or yellow warning screen which appears
+before the emulation starts, if any. Notice that if you have information which
+can help to improve the emulation of a supported system, or if you can directly
+contribute fixes and/or addition to the current source, you can do any of the
+following:
 
+    * Send in a pull request (for code) or open an issue (information) on our Github page.
+    * Post the information or code on the `MAME Forums <https://forum.mamedev.org/>`_
+    * Follow the instructions on our `contact page <https://www.mamedev.org/contact.html>`_
 
-Supported OS
-============
 
-The current source code can be directly compiled under all the main OSes: Microsoft Windows (both with DirectX/BGFX native support or with SDL support), Linux, FreeBSD, and Mac OS X. Also, both 32-bit and 64-bit are supported, but be aware 64-bit often shows significant speed increases.
+Supported OS
+------------
 
+The current source code can be directly compiled under all the main OSes:
+Microsoft Windows (both with DirectX/BGFX native support or with SDL support),
+Linux, FreeBSD, and macOS. Also, both 32-bit and 64-bit are supported, but
+be aware 64-bit often shows a significant speed increase over 32-bit.
 
 System Requirements
-===================
+-------------------
 
-MAME is written in fairly generic C/C++, and has been ported to numerous platforms. Over time, as computer hardware has evolved, the MAME code has evolved as well to take advantage of the greater processing power and hardware capabilities offered.
+MAME is written in C++, and has been ported to numerous platforms. Over time, as
+computer hardware has evolved, the MAME code has evolved as well to take
+advantage of the greater processing power and hardware capabilities offered.
 
-The official MAME binaries are compiled and designed to run on a standard Windows-based system. The minimum requirements are:
+The official MAME binaries are compiled and designed to run on a standard
+Windows-based system. The minimum requirements for Windows are:
 
 * Intel Core series CPU or equivalent, at least 2.0 GHz
 * 32-bit OS (Vista SP1 or later on Windows, 10.9 or later on Mac)
```

Comment:
Official binaries now require Core 2 or equivalent and a 64-bit OS (Vista or later required due to something in the MinGW runtime).

Reply:
My testing shows that Windows 7 is minimum for Windows (only SP1 tried). Vista doesn't work.

Reply:
Yeah, saw that a bit ago. Thanks for the heads-up. That's now corrected.